specification/subscription/resource-manager/Microsoft.Subscription/preview/2019-10-01-preview/subscriptions.json

```diff
@@ -617,6 +756,92 @@
           "type": "string"
         }
       }
+    },
+    "PutSubscriptionRequest": {
+      "description": "The parameters required to create a new subscription.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/PutSubscriptionRequestProperties",
+          "description": "Put subscription request properties."
+        }
+      }
+    },
+    "PutSubscriptionRequestProperties": {
+      "description": "Put subscription properties.",
+      "required": [
+        "displayName",
+        "skuId",
+        "billingScope"
+      ],
+      "properties": {
+        "displayName": {
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "skuId": {
+          "description": "The SKU ID of the Azure plan.",
+          "type": "string"
+        },
+        "billingScope": {
```

Comment:
Can this be an enum?

Reply:
cannot be enum, will be of below formats:
/billingAccounts/{billingAccountId}/customers/{customerId}
/billingAccounts/{billingAccountId}/billingProfiles/{billingProfileId}/invoiceSections/{invoiceSectionId}
is it a good idea to specify above formats in "billingScope" description?

Reply:
Thanks for clarifying. In that case, please just improve the description for this field - my assumption from reading the description was that it should be set to either "customerLed" or "fieldLed" etc.